### PR TITLE
code review for pr #529

### PR DESCRIPTION
**Pull request by mojavelinux**
- control toc in documents rather than globally
- keep user manual as standalone document
- use amber 4 from material for CTA button
- simplify styling of TOC; switch back to using background color
- consolidate styles
- tweak margin around control bar

### CODE REVIEW

File: docs/asciidoctorj.adoc [hunk 1]
```diff
@@ -39,7 +40,7 @@ ifdef::env-github[]
 image::http://img.shields.io/travis/asciidoctor/asciidoctorj/master.svg[Build Status,90,18,link=https://travis-ci.org/asciidoctor/asciidoctorj]
 endif::[]
 
-ifdef::env-site,env-browser[]
+ifeval::["{toc}" == "macro"]
```
Comment (mojavelinux) [inline docs/asciidoctorj.adoc]: This should be `ifeval::["{toc-placement}" == "macro"]`

File: docs/asciidoc-article.adoc [hunk 2]
```diff
@@ -2,7 +2,6 @@
 :description: This is an AsciiDoc article template. It illustrates the usage of the most common content structures and markup used in AsciiDoc documents and parsed by Asciidoctor, including section headings, lists, tables, example blocks, sidebars, image macros, listing blocks, literal blocks, quote blocks, verse blocks, admonitions, footnotes, links, and cross references.
 :keywords: AsciiDoc, Asciidoctor, AsciiDoc template, Asciidoctor template, AsciiDoc article template, Asciidoctor article template, AsciiDoc example, Asciidoctor example, AsciiDoc table example, AsciiDoc list example, AsciiDoc heading example, AsciiDoc article example, AsciiDoc sidebar example, AsciiDoc image example, AsciiDoc listing example, AsciiDoc code example, AsciiDoc literal example, AsciiDoc quote example, AsciiDoc verse example, AsciiDoc footnote example, AsciiDoc link example, AsciiDoc cross reference example
 :page-layout: docs
-:sectnums:
```
Comment (Mogztter) [inline docs/asciidoc-article.adoc]: You don't like section numbering ? I think this looks great with TOC (especially with subsections, see http://groovy-lang.org/differences.html)

File: docs/asciidoc-recommended-practices.adoc [hunk 3]
```diff
@@ -3,7 +3,6 @@ Dan Allen
 v1.0-HEAD, 2013-02-10
```
Comment (Mogztter) [inline docs/asciidoc-recommended-practices.adoc]: No TOC here ?

File: stylesheets/styles.scss [hunk 4]
```diff
@@ -1320,57 +1322,39 @@ li > span.separator {
...
-  display: inherit;
-}
-
-ul.toolbar.off > li > a.toggle-navigation > i.fa-toggle-on {
-  display: none;
-}
-
-ul.toolbar.off > li > a.toggle-navigation > i.fa-toggle-off {
-  display: inherit;
+ul.toolbar {
```
Comment (Mogztter) [inline stylesheets/styles.scss]: I need to learn Sass ;)
This is much much better!

File: docs/asciidoc-article.adoc (hunk 2 above)
Comment (mojavelinux) [inline docs/asciidoc-article.adoc]: I should have clarified. I'd like that to be part of a separate discussion/PR. It fits for some pages, but not others. We need to decide when section numbers should and shouldn't be used.

File: docs/asciidoc-recommended-practices.adoc (hunk 3 above)
Comment (mojavelinux) [inline docs/asciidoc-recommended-practices.adoc]: Oops, I dropped that one by accident.

File: stylesheets/styles.scss (hunk 4 above)
Comment (mojavelinux) [inline stylesheets/styles.scss]: There's one simple rule. If there's a space in the selector, you can nest it ;)

File: docs/asciidoctorj.adoc (hunk 1 above)
Comment (Mogztter) [inline docs/asciidoctorj.adoc]: Are you saying we need to change this line now or we should change this line (one day for "correctness")

File: docs/asciidoc-article.adoc (hunk 2 above)
Comment (Mogztter) [inline docs/asciidoc-article.adoc]: Ok

**Comment by Mogztter** [comment]
This looks good to me, I just want to run these changes locally before merging

File: docs/asciidoctorj.adoc (hunk 1 above)
Comment (mojavelinux) [inline docs/asciidoctorj.adoc]: I'm saying that I used the wrong attribute in my PR. It should be `toc-placement`, not `toc`.

**Pull request merged**